paper.md

```diff
@@ -0,0 +1,104 @@
+
+What some are now calling “classical” spectral element methods use tensor product bases on quadrilateral or hexahedral meshes. These bases lead to very efficient implementations and have high order quadratures that can be used to approximate the integrals found in weak forms of the equations. Of the widely available spectral element packages, SemTex, Sem2dPack, Nek5000, FLEXI, FLUXO, Trixi.jl, and HORSES3D primarily or exclusively use quadrilateral and hexahedral meshes.
+
+Unfortunately, unstructured meshes for quad/hex elements are difficult to generate even for low order finite elements [@Bommes:2013lc].
+The advantages not withstanding, a major impediment to the application of SEMs has been the availability of appropriate general purpose mesh generation software that can generate elements of arbitrary order, especially in open-source form. A survey of the literature, practitioners, and user manuals for available spectral element software packages highlights these difficulties. In 2002 Sherwin and Peiro [@Sherwin:2002vx] wrote:  "The development of robust unstructured high-order methods is currently limited by the inability to consistently generate valid computational meshes for complex geometries without user intervention." This has remained true particularly for quadrilateral and hexahedral meshes.
+
+For these reasons, HOHQMesh was developed to generate all-quadrilateral and extruded hexahedral meshes suitable for use with spectral element methods. HOHQMesh is a direct quadrilateral mesher, which generates quadrilateral elements by the subdivision method of Schneiders [@schneiders2000algorithms] rather than indirectly from a triangular mesh or by curving a low order mesh. It also sizes and curves the elements based on the length scales in the model, rather than try to modify an existing low order mesh.
+
+# Features
+
+HOHQMesh is designed to require minimal input from the user through the use of a control file. The model defines the geometry in terms of an outer and inner boundary curves.
+
+HOHQMesh features include:
+
+- Unstructured all-quadrilateral or hexahedral meshes
+- Isoparametric polynomial boundary approximations of arbitrary order
+- Automatic geometry-guided refinement
+- Optional user specified local refinement
+- Interior boundaries to separate regions of different properties
+- Symmetric mesh generation
+- Hexahedral meshes from extrusion, rotation, and sweeping of a quadrilateral mesh, with or without scaling
+- Bottom topography variations for extruded hexahedral meshes with automatic resolution of topographic features
```

Comment:
Suggest clarifying : "Bottom topography variations, defined through functional form or input topography data, for extruded hexahedral meshes..."